### PR TITLE
Rs 2022 07 01.1

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Arcueid/Day 4/04_00_ARC04_9.txt [hunk 1]
```diff
@@ -91,17 +96,17 @@ But, more importantly―――
...
 -- Page 693, Offset 7413.
 -- 「本当よ。だってわたし、血を吸うのが恐いんだもの」
-"I'm serious. Besides, sucking blood is really scary for me."
+"I'm serious. You see, the thought of sucking blood is really scary for me."
 }
 [sha:f94bc5563958851f81213566cf7cbca1fe7c94a3]{
 -- Page 694, Offset 7414.
 -- 「血を吸うのが恐いって、吸血鬼なのに？」
-"You're afraid of sucking blood, even though you're a vampire?"
+"Let me get this straight, you're a vampire but you're afraid of sucking blood?"
```
Comment (Avuxo) [inline script/Arcueid/Day 4/04_00_ARC04_9.txt:104]: vampire, but you're

File: script/Arcueid/Day 4/04_00_ARC04_9_2.txt [hunk 2]
```diff
@@ -121,37 +124,39 @@ I hear countless knocks on the door.
...
 -- Page 944, Offset 8177.
 -- ―――この部屋に残った事が失策なら、
-―――If staying in this room was a mistake,
+If staying in this room was a mistake―――
 }
 [sha:11c301a2d3c5792bb9a772ae8a1e56ddc92d9614]{
 -- Page 944, Offset 8178.
 -- そんな質問をしたのも失策だったのか。
-then was it also a mistake to ask such a question?
+// 質問 here the immediately prior shitsumon, or the question of should I stay or should I go now?
```
Comment (Avuxo) [inline script/Arcueid/Day 4/04_00_ARC04_9_2.txt:152]: i _think_ it's the former

File: script/Arcueid/Day 5/05_00_ARC05_9.txt [hunk 3]
```diff
@@ -268,7 +268,7 @@ That's backwards. If that's your goal, you need to do the opposite.
 [sha:d3315454c22a82354d634a53b5bba0504b724f06]{
 -- Page 575, Offset 10158.
 -- アレは容認してはいけないもの。
-// assuming 「アレ」 is broob's very existence and not 「逃げる」 (その理由) 
+// assuming 「アレ」 is broob's very existence and not 「逃げる」 (その理由)
```
Comment (Avuxo) [inline script/Arcueid/Day 5/05_00_ARC05_9.txt:271]: changes in these files were made in previous pr too?

**Review by Avuxo — APPROVED** [review]
4_9_2 was very immersive because its 89f and 87% humidity :sacchincopium:

File: script/Arcueid/Day 5/05_00_ARC05_9.txt (hunk 3 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 5/05_00_ARC05_9.txt:271]: Yeah whichever goes in first will just win. They're just from re-exporting the entire script (cleaning comments, whitespace, etc)

**Pull request merged**